app/controllers/albums.js

```diff
@@ -36,3 +36,21 @@ exports.buyAlbum = async (req, res, next) => {
     next(err);
   }
 };
+
+exports.listPurchasedAlbums = async (req, res, next) => {
+  try {
+    const albumUser = await Album.getAllAlbumBy({
+      userId: req.params.userId
+    });
+
+    const promises = albumUser.map(async x => {
```

Comment:
Check declaration of things, `promises`- `x` are a little bit tricky of read and understand

Reply:
is correct, by mistake I corrected it in another branch